src/main/webapp/dev/js/common/questionRank.js

```diff
@@ -231,6 +231,25 @@ function hideInvalidRankRecipientFeedbackPaths(qnNum) {
     }
 }
 
+function prepareRankedQuestionCheckbox() {
+    $(`[id^=${ParamsNames.FEEDBACK_QUESTION_RANKMAXOPTIONSCHECKBOX}]`
```

Comment:
So I'm wondering why match all questions when we can match the `newQuestionNum` which is `-1`?

Reply:
Because this is not for new questions only. This needs to work for all questions. Say, the instructor decided to edit a previous question, this interaction needs to work as well?

Reply:
My bad. I missed that completely.